packages/gatsby/cache-dir/static-entry.js

```diff
@@ -8,6 +8,7 @@ const { get, merge, isObject, flatten, uniqBy } = require(`lodash`)
 const apiRunner = require(`./api-runner-ssr`)
 const syncRequires = require(`./sync-requires`)
 const { dataPaths, pages } = require(`./data.json`)
+const { version: gatsbyVersion } = require(join(process.cwd(), 'node_modules/gatsby/package.json'))
```

Comment:
I wonder if we can just do ``require(`gatsby/package.json`)``?

Reply:
We sure can! Thank you :)